### DISCUSSION TITLE
I love the press and hold pop-up menu feature … BUT …

### DISCUSSION THREAD

**Discussion by jjweichert** [javikr/sofatime#56] [Ideas]
Currently there’s a very handy press and hold pop-up menu available in the ‘View all’ screen inside of the ‘Watch’ menu. For example, inside ‘View all’ I can press and hold on any show and then quickly mark as watched or archive, etc. I love this feature!! BUT it doesn’t work on the default ’Watch’ page/view! Why not? Could/would you please enable this existing feature on the default view as well??

**Comment by javikr** [comment]
Hi there! 

This feature was added in 2.38.0. I will close this issue. Thanks!
